octoprint_mrbeam/iobeam/dust_manager.py

```diff
@@ -150,6 +160,21 @@ def _onEvent(self, event, payload):
 		elif event == OctoPrintEvents.SHUTDOWN:
 			self.shutdown()
 
+	def _finish_test_fan_rpm(self):
+		# Write to analytics if the values are valid
```

Comment:
pls wrapp in try/except since this is a separate thread.

Reply:
done